docker/.cico.pipeline

```diff
@@ -0,0 +1,131 @@
+def c7arches = ["ppc64le", "x86_64", "aarch64"]
+def stages = [:]
+
+def onmyduffynode(script)
+{
+    ansiColor('xterm')
+    {
+        timestamps
+        {
+            sh 'ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -l root ${DUFFY_NODE}.ci.centos.org -t \"export REPO=${REPO}; export BRANCH=${BRANCH};\" "' + script + '"'
+        }
+    }
+}
+
+def syncfromduffynode(rsyncpath)
+{
+    sh 'rsync -e "ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -l root " -Ha --include=' +  rsyncpath +  " ${DUFFY_NODE}.ci.centos.org:~/ ./"
+}
+
+
+
+def synctoduffynode(source)
+{
+    sh 'scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -r ' + source + " " + "root@" + "${DUFFY_NODE}.ci.centos.org:~/"
+}
+
+
+node('sig-cloudinstance'){
+    stage('make placeholder'){
+        env.time_stamp = sh(
+                script: "date +%y%m",
```

Comment:
can we chance the  centos-7.${time_stamp} to -${time_stamp?} otherwise it looks a bit too similar to the release versioning, which wouldn't be correct.